src/main/java/io/cdap/plugin/aws/s3/source/S3BatchSource.java

```diff
@@ -227,5 +246,44 @@ Map<String, String> getFilesystemProperties() {
       }
       return GSON.fromJson(fileSystemProperties, MAP_STRING_STRING_TYPE);
     }
+
+    public boolean shouldVerifyCredentials() {
+      return verifyCredentials == null ? false : verifyCredentials;
```

Comment:
since `verifyCredentials` is also macro enabled it should be something like:

```
return containsMacro("verifyCredentials") || verifyCredentials == null ? false : verifyCredentials;
```

Reply:
The credentials validation was intended to be done at the design time. So having the flag as macro doesn't make much sense to me. Shall I remove the field as Macro? What do you suggest?

Reply:
sounds good to me.

Reply:
Removed the macro check. aaad180621b860d417a288c466d56e024ac9e68a